Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin-release:golang-1.12 as builder
+FROM openshift/origin-release:golang-1.13 as builder
```

Comment:
Alas, this same change is pending in the 1.18 rebase PR.

Reply:
Are there blockers on that? Can we push it through first? 

Without this change, the controller-runtime client fails because it requires go 1.13.

Reply:
My plan was to rebase it after friday. There's nothing blocking it, but pretty much all PRs will need to be rebased (and they're non-trivial since APIs changed)